BuilderDesignPattern/BuilderDesignPattern/Program.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BuilderDesignPattern
+{
+    class Program
+    {
+        public interface IPizzaBuilder
+        {
+            void Reset();
+            void BuildDough();
+            void BuildSauce();
+            void BuildTopping();
+            void BuildCheese();
+            void BuildDoubleCheese();
+        }
+
+        public class WheatPizzaBuilder : IPizzaBuilder
+        {
+            private Pizza pizza = new Pizza();
+
+            public void Reset()
+            {
+                this.pizza = new Pizza();
+            }
+
+            public void BuildDough() 
+            {
+                this.pizza.Add("Add wheat dough");
+            }
+
+            public void BuildSauce() 
+            {
+                this.pizza.Add("Add sauce");
+            }
+
+            public void BuildTopping() 
+            {
+                this.pizza.Add("Add topping");
+            }
+
+            public void BuildCheese()
+            {
+                this.pizza.Add("Add cheese");
+            }
+
+            public void BuildDoubleCheese()
+            {
+                this.pizza.Add("Add double cheese");
+            }
+
+            public Pizza GetPizza()
+            {
+                Pizza result = this.pizza;
+                this.Reset();
+                return result;
+            }
+
+        }
+
+        public class WhitePizzaBuilder : IPizzaBuilder
+        {
+            private Pizza pizza = new Pizza();
+
+            public void Reset()
+            {
+                this.pizza = new Pizza();
+            }
+
+            public void BuildDough()
+            {
+                this.pizza.Add("Add white dough");
+            }
+
+            public void BuildSauce()
+            {
+                this.pizza.Add("Add sauce");
+            }
+
+            public void BuildTopping()
+            {
+                this.pizza.Add("Add topping");
+            }
+
+            public void BuildCheese()
+            {
+                this.pizza.Add("Add cheese");
+            }
+
+            public void BuildDoubleCheese()
+            {
+                this.pizza.Add("Add double cheese");
+            }
+
+            public Pizza GetPizza()
+            {
+                Pizza result = this.pizza;
+                this.Reset();
+                return result;
+            }
+        }
+
+        public class Pizza
```

Comment:
Good job declaring the `Pizza` class but even though your solution is correct, it should have been two classes as written in the requirements.